packages/cms-data-sync/src/utils/setup.ts

```diff
@@ -18,6 +18,9 @@ import { rateLimiter } from './rate-limiter';
 export const isVerbose = () =>
   process.env.VERBOSE_DATA_SYNC && process.env.VERBOSE_DATA_SYNC === 'true';
 
+export const upsertInPlace =
+  process.env.UPSERT_IN_PLACE && process.env.UPSERT_IN_PLACE === 'true';
```

Comment:
Not essential, but it would be nice to be able to easily specify it via a CLI flag when running locally.

```suggestion
  (process.env.UPSERT_IN_PLACE && process.env.UPSERT_IN_PLACE === 'true') ||
  process.argv.includes('--upsert');
```